src/recommendations/pipe/recommendations-validatation.pipe.ts

```
@@ -0,0 +1,8 @@
+import { ArgumentMetadata, Injectable, PipeTransform } from '@nestjs/common';
+
+@Injectable()
+export class RecommendationValidataionPipe implements PipeTransform {
+  transform(value: any, metadata: ArgumentMetadata) {
```

ESLint

[error] 'value' is defined but never used.

(@typescript-eslint/no-unused-vars)